src/human-practices/education/lets-talk-science.md

```diff
@@ -0,0 +1,28 @@
+# Let's Talk Science
+
+## Overview
+The collaboration between Let’s Talk Science and the UBC iGEM team is an educational outreach initiative designed to engage high school students in the exciting field of synthetic biology. Through interactive workshops, seminars, and hands-on activities, this initiative aims to foster greater interest in the field of synthetic biology and inspire the next generation of scientists. 
+
+## Context and Scope
+Let’s Talk Science’s extensive experience in science education combined with our UBC iGEM team’s research and practical knowledge in synthetic biology is ideal for sparking interest in the topic of synthetic biology for high school students. This program will focus on making advanced scientific concepts accessible and engaging to young learners across various educational backgrounds. 
+
+## Goals and non-goals
+### Goals
+To increase awareness and understanding of synthetic biology among highschool students.
+To inspire students to pursue further education and careers in the STEM fields, particularly in synthetic biology and related disciplines. 
+
+### Non-Goals
+To target audiences outside of the highschool education level for this specific initiative
+To provide university level technical training
+
+## Plan
+Brainstorming and ideation of classroom activities
+Pitching idea to LTS representatives 
+Marketing and Branding
+Contacting highschools through Let’s Talk Science 
+Volunteer Training for our internal team
+Final activity preparations + material gathering
+Executing our program with classroom visits
```

Comment:
To make this a list, add `-` before each point:

```
- point 1
- point 2
```

you will get 

- point 1
- point 2